lib/transport/net_transport.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var AbstractTransport = require('./abstract_transport.js');
+var util = require("util");
```

Comment:
'util' not "util"

There, I won the award for the most nitpicky code-review comment!